src/log.cpp

```diff
@@ -52,5 +57,9 @@ void librealsense::log_to_callback(rs2_log_severity min_severity, log_callback_p
 void librealsense::reset_logger()
 {
 }
+
+void librealsense::enable_rolling_files(std::size_t max_size )
+{
+}
```

Comment:
I think it needs to throw

Reply:
why? if so, shouldn't all the other functions also throw?

Reply:
done